### PR TITLE
Bring sample readme updates to preview

### PR DESCRIPTION
**Pull request by abhipsaMisra**
PR to bring sample readme updates to `preview`. The changes in the .cs files are all comments-related changes.

### CODE REVIEW

File: e2e/test/iothub/FileUploadE2ETests.cs [hunk 1]
```diff
@@ -130,10 +130,7 @@ private async Task UploadFileGranularAsync(Stream source, string filename, Http1
                 cert = s_selfSignedCertificate;
                 x509Auth = new DeviceAuthenticationWithX509Certificate(testDevice.Id, cert);
 
-                // The X509 certificate being used for device authentication needs to be set into FileUploadTransportSettings as well,
-                // so that the HttpClient created for file upload operation has access to those certificates.
-                clientOptions.FileUploadTransportSettings.ClientCertificate = cert;
-                deviceClient = DeviceClient.Create(testDevice.IoTHubHostName, x509Auth, Client.TransportType.Http1, clientOptions);
```
Comment (abhipsaMisra) [inline e2e/test/iothub/FileUploadE2ETests.cs:136]: this was updated in master

**Review by vinagesh — APPROVED** [review]

File: e2e/test/iothub/FileUploadE2ETests.cs (hunk 1 above)
Comment (vinagesh) [reply e2e/test/iothub/FileUploadE2ETests.cs:136]: This is for the new Fairfax support. Is it directly merged to preview?

Comment (abhipsaMisra) [reply e2e/test/iothub/FileUploadE2ETests.cs:136]: this is sequence followed here:

master -> previews/pnpApi
update previews/pnpApi
previews/pnpApi -> preview

Comment (abhipsaMisra) [reply e2e/test/iothub/FileUploadE2ETests.cs:136]: We had this additional x509 certificate initialization for file upload operations, as a part of our e2e tests. This is not required and se the test was updated in master.

**Pull request merged**